modules/installation-initializing.adoc

```diff
@@ -207,15 +207,10 @@ $ ./openshift-install create install-config --dir <installation_directory> <1>
 <1> For `<installation_directory>`, specify the directory name to store the
 files that the installation program creates.
 +
-[IMPORTANT]
-====
-Specify an empty directory. Some installation assets, like bootstrap X.509
-certificates have short expiration intervals, so you must not reuse an
-installation directory. If you want to reuse individual files from another
-cluster installation, you can copy them into your directory. However, the file
-names for the installation assets might change between releases. Use caution
-when copying installation files from an earlier {product-title} version.
-====
+When specifying the directory:
```

Comment:
@jhixson74 Prior to Matthew leaving, he asked that I update the installation doc to state that the installation directory now requires execute permissions. Requesting your review. Thanks.